### ISSUE TITLE
Problem compiling using TexLive 2015

### ISSUE THREAD

**Issue by brosenan** [programming-journal/programming#2]
In preparation of the camera ready version of  our paper we tried to compile our paper against the latest version of the template, but it did not compile due to a reference to a missing macro: **\setcapdynwidth**. This macro is defined in the latest version of KomaScript, but not in the one distributed as part of TexLive2015 (the TexLive version the template requires).

The version we used:
```bash
~$ latex --version
pdfTeX 3.14159265-2.6-1.40.16 (TeX Live 2015/Debian)
kpathsea version 6.2.1
Copyright 2015 Peter Breitenlohner (eTeX)/Han The Thanh (pdfTeX).
There is NO warranty.  Redistribution of this software is
covered by the terms of both the pdfTeX copyright and
the Lesser GNU General Public License.
For more information about these matters, see the file
named COPYING and the pdfTeX source.
Primary author of pdfTeX: Peter Breitenlohner (eTeX)/Han The Thanh (pdfTeX).
Compiled with libpng 1.6.17; using libpng 1.6.17
Compiled with zlib 1.2.8; using zlib 1.2.8
Compiled with poppler version 0.41.0
```

**Comment by krono** [comment]
Fixed with #5.

**Comment by krono** [comment]
Quick fix for submission: put this before `\documentclass`

```LaTeX
\providecommand*\setcapdynwidth{}
% here follows \documentclass
```
